### ISSUE TITLE
Make new "chatmail unencrypted error" info message clickable

### ISSUE THREAD

**Issue by Simon-Laux** [deltachat/deltachat-desktop#3644]
this requires a core with https://github.com/deltachat/deltachat-core-rust/pull/5195

info message type can be detected by SystemMessageType (already done for webxdc messages and guaranteed e2ee info messages). - (link2xt commented on the core pr that it is not available in deltachat.h yet, but it is already available in jsonrpc, so desktop can already do this)

## Tasks

- [x] set new stock string
- [ ] update local help to a version that includes information about chatmail not allowing unencrypted messages
- [x] detect click on this new variant of info messages and open local help at the right anchor location

**Comment by r10s** [comment]
i am on that on android as well, let's wait until strings are settled there

**Comment by Simon-Laux** [comment]
also the help page does not include the section about this yet, should I make an issue or is this already tracked somewhere else?

**Comment by r10s** [comment]
i finished the [android pr](https://github.com/deltachat/deltachat-android/pull/2940), once that is merged, there are to new strings available, `invalid_unencrypted_tap_to_learn_more` and `invalid_unencrypted_explanation`.

the dialog shows the "Scan QR code" and "Help" button as known from guarantee-broken and cannot-add-unverified; as the user's problem at hand is the scanning, the current help is pretty much on point. at some point, we can mention chatmail, alternatives to QR code scanning and refine otherwise, but that is no prerequisite
